The_Glowing_Volcano/Assets/Scripts/MenuSound.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+[RequireComponent(typeof(Button))]
+public class MenuSound : MonoBehaviour
+{
+    public AudioClip sound; 
```

Comment:
@ThreeEyedVenelope @ifaith1 use serialize field and not public for exposing fields to the editor